source/server-discovery-and-monitoring/tests/errors/non-stale-network-timeout-error.yml

```diff
@@ -5,6 +5,7 @@ phases:
   responses:
   - - a:27017
     - ok: 1
+      helloOk: true
```

Comment:
The following tests appear to be missing `helloOk` in some mocked responses:
   - `tests/monitoring/required_replica_set.yml`
   - `tests/monitoring/standalone_suppress_equal_description_changes.yml`
   - `tests/monitoring/required_replica_set.yml`
   - `tests/monitoring/standalone.yml`

Should those be updated?

Edit: I believe the rationale for _not_ updating those is because they mock a `maxWireVersion: 4`. No need to change if that is the case.

Reply:
No, this was an accidental omission on my part. I had my working directory set incorrectly in vim, which resulted in my grep missing files in other folders. Fixing this now.